classes/tweet.rb

```diff
@@ -1,4 +1,47 @@
 class Tweet
-  attr_accessor :content, :location, :created_at, :zombie, :tweet_id
+  attr_accessor :content, :location, :created_at,
+                        :zombie, :unique_id, :favs, :retweets,
+                        :original_tweet, :retweeted
+
+  # Class level attributes
+  @@tweet_id = 1
+  @@all_tweets = []
+
+  # Run this block on startup
+  def initialize
+    # Assign a unique id to each tweet and increment the counter
+    @unique_id = @@tweet_id
+    @@tweet_id += 1
```

Comment:
Instead of relying on something like `@@tweet_id` to give you a unique ID, there are a number of other things you can do:
* check `@@all_tweets.size` to get the current size and then add 1
* `require 'securerandom'` and then call `SecureRandom.hex` to get a unique ID